shared/chat/conversation/info-panel/container.js

```diff
@@ -156,24 +157,33 @@ class InfoPanelSelector extends React.PureComponent<Props> {
     return isMobile ? (
       <ConnectedInfoPanel onBack={this.props.onBack} conversationIDKey={this.props.conversationIDKey} />
     ) : (
-      <Box onClick={this.props.onBack} style={clickCatcherStyle}>
-        <Box style={panelContainerStyle} onClick={evt => evt.stopPropagation()}>
-          <ConnectedInfoPanel onBack={this.props.onBack} conversationIDKey={this.props.conversationIDKey} />
-        </Box>
-      </Box>
+      <Kb.Box onClick={this.props.onBack} style={clickCatcherStyle}>
```

Comment:
this only shows desktop, we should have some example of mobile working also

Reply:
I wasn't sure what trivial thing to animate on mobile.